posts/parent-class/index.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Choosing the Right Parent for R Object Classes"
```

Comment:
I think this title only partly captures the scope of the post.

I would suggest something like "Recommendations and good practices for creating custom S3 objects and methods in R".

Reply:
I think the suggestion is now not precise enough as we are likely to have other posts on specific good practices for S3. This is a specific areas of good practices for S3.